pkg/database-types/database_type.go

```diff
@@ -15,10 +15,11 @@ type DatabaseType interface {
 	OMTypeName() string
 
 	// translate the connection information from the Fybrik format to the OM format
-	TranslateFybrikConfigToOpenMetadataConfig(map[string]interface{}, *string) map[string]interface{}
+	TranslateFybrikConfigToOpenMetadataConfig(map[string]interface{}, string, *string) map[string]interface{}
```

Comment:
there is no argument names, I'd suggest adding some explanation in the comments, what are the arguments. Currently, it's `string` and `*string`, what are they, why one is string and another *string ...